Multisig/UI/Transaction/TransactionListViewController/Cells/TransactionListTableViewCell.swift

```diff
@@ -35,6 +35,12 @@ class TransactionListTableViewCell: SwiftUITableViewCell {
         confirmationsCountLabel.setStyle(.footnote2)
     }
 
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        typeImageView.layer.cornerRadius = 0
+        typeImageView.clipsToBounds = false
```

Comment:
I'd prefer to have the image processed - please see the UIImageViewExtension.swift line 20 for example

The same is relevant for the line 59 of this file as well

Reply:
I tried to use RoundCornerImageProcessor, but it does not work for placeholder image. So the option was to round UIImage that is passed to placeholder, or to round UIImageView. I selected the second option.
Or do you mean smth different?

Reply:
oh I see